glean-core/ios/GleanTests/TestUtils.swift

```diff
@@ -50,47 +50,12 @@ func stubServerReceive(callback: @escaping (String, [String: Any]?) -> Void) {
 /// This also prevents outgoing network requests during unit tests while
 /// still allowing us to use the default telemetry endpoint.
 func resetGleanDiscardingInitialPings(testCase: XCTestCase, tag: String, clearStores: Bool = true) {
-    let expectation = testCase.expectation(description: "\(tag): Ping Received")
```

Comment:
Why remove this part of the test? I mean, I know we test this in glean-core also but was the change you made interfering with this?

Reply:
We have no initial pings anymore!
Because now we're only sending the first baseline ping on an `active` app, which tests don't have (there's no UI -> no `didBecomeActiveNotification` -> no ping).
So keeping this in now breaks tests, because it waits for pings that never arrive.